src/timeline/EventStore.h

```diff
@@ -66,7 +66,8 @@ class EventStore : public QObject
         // relatedFetched event
         mtx::events::collections::TimelineEvents *get(std::string_view id,
                                                       std::string_view related_to,
-                                                      bool decrypt = true);
+                                                      bool decrypt       = true,
+                                                      bool resolve_edits = true);
         // always returns a proper event as long as the idx is valid
         mtx::events::collections::TimelineEvents *get(int idx, bool decrypt = true);
```

Comment:
Does this function need to have the resolve_edits changes incorporated as well?

Reply:
Not really, imo. This always resolves the edits, the other function just alloes side stepping that, to prevent loops as wel es show the edit history in the future.